include/phasar/Utils/TypeTraits.h

```diff
@@ -21,84 +21,94 @@ namespace psr {
 namespace detail {
 
 template <typename T, typename = void>
-struct is_iterable : public std::false_type {};
+struct is_iterable : public std::false_type {}; // NOLINT
 template <typename T>
-struct is_iterable<T, std::void_t<typename T::const_iterator,
+struct is_iterable<T, std::void_t<typename T::const_iterator, // NOLINT
                                   decltype(std::declval<T>().begin()),
                                   decltype(std::declval<T>().end())>>
     : public std::true_type {};
 
-template <typename T> struct is_pair : public std::false_type {};
+template <typename T> struct is_pair : public std::false_type {}; // NOLINT
 template <typename U, typename V>
-struct is_pair<std::pair<U, V>> : public std::true_type {};
+struct is_pair<std::pair<U, V>> : public std::true_type {}; // NOLINT
```

Comment:
Again, what are these errors? This is normal valid code, this should not cause a warning.